src/main/kotlin/org/agoranomic/assessor/decisions/8253-8265.kt

```diff
@@ -347,5 +347,21 @@ Enact a new Rule of Power 3.0 with the text:
             PRESENT on 8264
             FOR on 8265 // Only proposal not listed in message
         }
+
+        votes(Gaelan) {
+            FOR on 8253
+            FOR on 8254
+            FOR on 8255
+            AGAINST on 8256
+            // TODO: conditional vote on 8257: if 8259 would be ADOPTED, AGAINST; otherwise, FOR
```

Comment:
I'm so sorry.

Reply:
It's fine, that's a perfectly sensible vote. And it's nowhere near the most complicated vote I've seen so far: https://mailman.agoranomic.org/cgi-bin/mailman/private/agora-business/2019-August/041171.html.